scripts/01_directional_clustering.py

```diff
@@ -257,34 +260,30 @@
 # below there is the simplest snippet, but you can see more stuff
 # in the scripts/visualization folder
 
-# ClusterPlotter is a custom wrapper around a COMPAS MeshPlotter
-# the COMPAS MeshPlotter is built atop of pure Matplotlib (which is crazy)
-# what is different here is that I extended the plotter so that it can plot
-# vector fields directly as little lines via
-# ClusterPlotter.draw_vector_field_array()
-plotter = ClusterPlotter(mesh, figsize=(12, 9))
+# PlyPlotter is a custom wrapper around a Plotly graph object (Figure)
+# that handles formating and adjusts data structure.
+plotter = PlyPlotter() # plotly
```

Comment:
Nice to learn, thanks! But I'm removing the comment since this is actually trash from before when I was identifying my added code as `# plotly` and the previous code with `# matplotlib`